src/glcontext_html5.cpp

```diff
@@ -71,7 +71,7 @@ namespace bgfx { namespace gl
 
 		const char* canvas = (const char*) g_platformData.nwh;
 
-		EMSCRIPTEN_WEBGL_CONTEXT_HANDLE context = (EMSCRIPTEN_WEBGL_CONTEXT_HANDLE) g_platformData.context;
+		EMSCRIPTEN_WEBGL_CONTEXT_HANDLE context = (EMSCRIPTEN_WEBGL_CONTEXT_HANDLE)(uintptr_t) g_platformData.context;
```

Comment:
Actually use `bx::narrowCast` (it's recently added, for this kind of casting).

Reply:
Try:

```cpp
EMSCRIPTEN_WEBGL_CONTEXT_HANDLE context = bx::narrowCast<EMSCRIPTEN_WEBGL_CONTEXT_HANDLE>(uintptr_t(g_platformData.context) );
```

Reply:
Basically `void*` to `uintptr_t` but then `bx::narrowCast` does test that value passed is not truncated, and fails/asserts at that spot where issue occurred, and not at some later point.